### PR TITLE
fix #40827 vipboxes.eu

### LINKED ISSUE #40827: vipboxes.eu
### Issue URL (Annoyance)
[http://vipboxes.eu/moto_gp/vip-381023-formula-1-singapore-grand-prix-practice-1#](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fvipboxes.eu%2Fmoto_gp%2Fvip-381023-formula-1-singapore-grand-prix-practice-1%23)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Filters: | Filter unblocking search ads and self-promotion,<br>AdGuard Base

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/40827

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -1732,6 +1732,8 @@ mobilapk.com#%#AG_onLoad(function() { setTimeout(function() {document.getElement
 !
 ! https://forum.adguard.com/index.php?threads/35054/
 mi-globe.com#$# div > span[id^="ezoic-pub-ad-placeholder-"] + span[style^="display:block"] { position: absolute!important; left: -3000px!important; }
+! https://forum.adguard.com/index.php?threads/40827
+vipboxes.eu#$#body > div.show { visibility: hidden!important; }
```
Comment (BlazDT) [inline EnglishFilter/sections/general_extensions.txt]: Not shown in code.

File: EnglishFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -602,6 +602,7 @@ rarbgproxied.org,rarbg.to,rarbg-to.proxydude.xyz,rarbg-to.pbproxy.red,rarbg.unbl
 alluc.ee#?##resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 !
 spankwire.com#?##js-react-video-player > div > div > div:has(> a[href="https://www.czechsexcasting.com/"])
+vipboxes.eu#?#.row > div[class]:has(a[target="_blank"] > img)
```
Comment (BlazDT) [inline EnglishFilter/sections/css_extended.txt]: Not shown in code.

**Review by BlazDT — CHANGES_REQUESTED** [review]
Left-over at http://vipboxes.eu/moto_gp

#rightcolumnwrap div[id^="slot"][style]

**Comment by zloyden** [comment]
@BlazDT please take a look.
<details><summary>Screenshots:</summary>

image
image1
</details><br/>

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (BlazDT) [reply EnglishFilter/sections/general_extensions.txt]: Why not ##div[id^="banner"]?

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt]: > Why not ##div[id^="banner"]?

It covers only banner on the left side, not overlay on my side.

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**